src/tasks/WasmAppBuilder/EmccExec.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Tasks;
+
+/// <summary>
+/// Wrapper around 'emcc' exec command that enrich ouput with msbuild configuration for produced errors.
+/// </summary>
+public class EmccExec : Exec
```

Comment:
@radical Wrapping the `Exec` task is the best way I found for appending something to the exec output and leave the itself to stop the build.
Do you know of a better solution?

Reply:
Could you explain why this is needed?

Reply:
Wrapping `Exec` is fine. We do it in `RuntWithEmSdkEnv` task also.

Reply:
If emscripten produces an error for undefined symbol, we want should an additional message with information how to allow undefined symbols.

Reply:
Replaced with `OnError` msbuild feature.